src/scss/components/optional/_content-header.scss

```diff
@@ -24,13 +24,13 @@
             display: inline-block;
             margin: 0 spacing(x2) 0 0;
             font-weight: $font-weight-base;
-            @include font-size(base--scaleUp, false);
+            @include font-size(body-l, false);
         }
 
         .c-contentHeader-subTitle {
             display: block;
             margin-top: 4px;
-            @include font-size(base, false);
+            @include font-size(body-s, false);
```

Comment:
Similar to above - except could this `subTitle` be `heading-s` ?

Reply:
Have added a TODO to this to make sure this is addressed when we port the component (as it should have a different alias token at that point)